tests/scenarios/github_test_12/cmd1

```diff
@@ -0,0 +1,7 @@
+source: ubuntu-1-deployment
+cmd: curl 142.250.193.46
```

Comment:
specific IP? why do we need to use it?
is it to avoid a UDP connection for DNS resolution?

Reply:
> is it to avoid a UDP connection for DNS resolution?

Yes